app/assets/scss/_publication.scss

```diff
@@ -1,383 +1,4 @@
-        border-spacing: 0;
-        margin-bottom: $gutter*2;
-        th, td {
-          @include core-14;
-          background-color: transparent;
-          vertical-align: top;
-          text-align: left;
-          padding: $gutter-one-third $gutter-two-thirds $gutter-one-third 0;
-          border-bottom: 1px solid $grey-2;
-          border-width: 0 0 1px 0;
-        }
-        th {
-          font-weight: bold;
-        }
-        td {
-          p, ul, ol {
-            margin: 0;
-          }
-          p+p, p+ul, p+ol,
-          ul+p, ul+ol,
-          ol+p {
-            margin-top: $gutter-one-third;
-          }
-        }
-      }
-      .contact {
-        @extend %contain-floats;
-        background: $panel-colour;
-        margin: $gutter $gutter-half*-1;
-        padding: $gutter-one-third $gutter-half;
-        position: relative;
-
-        h4 {
-          @include core-19;
-          font-weight: bold;
-          margin: $gutter-one-third 0 0;
-        }
-        p {
-          @include copy-19;
-          margin: 0;
-        }
-        .adr,
-        .email-url-number,
-        .comments {
-          @include media(tablet) {
-            width: $half;
-            float: left;
-          }
-        }
-        .email-url-number {
-          p {
-            margin: 0;
-          }
-          span {
-            display: block;
-          }
-        }
-        .comments {
-          @include core-16;
-        }
-      }
-      .call-to-action {
-        margin: $gutter 0 $gutter;
-        background: $panel-colour none no-repeat 98% $gutter-two-thirds;
-        padding: $gutter-one-third ($gutter*2) $gutter-one-third $gutter-half;
-        position: relative;
-
-        h4 {
-          margin-top: 0;
-        }
-        p {
-          margin: 0;
-          padding: 0;
-        }
-        ul, ol {
-          margin-left: $gutter-two-thirds;
-          list-style-position: outside;
-        }
-        h3 {
-          padding: $gutter-one-sixth 0;
-          margin-top: 0px;
-          .number {
-            background: none;
-            @include media(tablet) {
-              width: $gutter-one-third*9;
-              margin-left: $gutter-one-third*-9;
-            }
-          }
-        }
-      }
-
-      .fraction {
-        sup,
-        sub {
-          @include core-14;
-        }
-        img {
-          display: inline-block;
-          width: auto;
-          margin: 0 0 -5px 0;
-        }
-      }
-    }
-  }
-  .back-to-content {
-    position: fixed;
-    bottom: $gutter-half;
-    a {
-      display: block;
-
-      &:before {
-        content: "\2191 ";
-      }
-    }
-  }
-  .js-back-to-content-stop {
-    clear: both;
-  }
-  p.report-a-problem-toggle a {
-    @include core-16;
-    color: $secondary-text-colour;
-    text-decoration: underline;
-  }
-  .report-a-problem-container {
-    margin-left: 0;
-  }
-
-}
-
-.floated-children {
-  overflow: hidden;
-}
 
 .terms-and-conditions {
```

Comment:
At this point it feels like this file just needs renaming to e.g. `_terms_and_conditions.scss`, given that's all we have left here.

We need to be careful with this, I think. If we start gutting parts of files, it feels like we're going to create more splintering rather than less, with different fragments of files in each app and no easy way to compare them anymore. This feels like it could increase the mental overhead for developers when moving between apps rather than reduce it.

Reply:
Good point.

I think it's generally OK if a file contains two 'components' or unrelated things are you remove one but we probably want to think a bit harder about removing parts of a 'component'.

I'll have a scan through myself but are there any in this PR you think might be worth rethinking?

Reply:
This was the main one. If we have files with multiple components in, and we are only using a subset of those components for one of our apps, then clearly they aren't that closely related and could be separate files. It feels like it would be simpler if our files were the same across apps, and we just include whichever files we need. That we, we can see (eg) `publication.scss` and know that it does the same thing in the buyer app and the briefs app. If they both do different things across apps, that's surely more confusing (to me at least)..

Reply:
I get you.

I've renamed the file and also renamed it to be more appropriate for the cookies page too. Let me know what you think.